src/graph/interface/op_schema.hpp

```diff
@@ -167,11 +167,13 @@ class op_schema_t {
     op_schema_t &set_output(size_t out_offset, std::string &&out_name,
             std::string &&dtype_string = "any");
 
-    /*! @brief Enable commutative inputs */
-    op_schema_t &set_commutative_inputs();
+    /*! @brief Enable commutative inputs for specific inputs*/
+    op_schema_t &set_commutative_inputs(
+            const std::set<size_t> &inputs = {0, 1});
```

Comment:
How many places is this function called? If possible, let's remove the default argument and add it explicitly at where it's needed.

Reply:
There are only four usages relying on the default argument. Removing it would indeed make the function interface clearer and more explicit.

Reply:
Updated